Stormshield/_meta/manifest.yml

```diff
@@ -0,0 +1,6 @@
+uuid: 758103e9-eb62-4b0d-8c71-0ff9c60ca677
+name: Stormshield
+slug: stormshield
+description: When you choose Stormshield to protect your critical and operational infrastructures, you’re building a relationship of trust with men and women who are passionate about what they do. Stormshield the European cybersecurity choice.
+data_sources:
+  Stormshield Network Security: The Stormshield Network Security range of firewalls combines performance and scalability to support your evolving IT and operational infrastructures.
```

Comment:
You should provide MITRE ATT&CK data sources. The complete list of supported datasources is available here: https://github.com/SekoiaLab/platform/blob/3fe48c4ac3cd4dcf5e769215fcd121b16f6a273f/resources/scripts/telltaler/telltaler/tips/intakes/intakes_datasources_are_known.py#L15-L81